less/home.less

```diff
@@ -0,0 +1,277 @@
+.home {
+  .jumbotron {
+    position: relative;
+
+    img {
+      width: 100%;
+    }
+
+    h1 {
+      position: absolute;
+      left: 60px;
+      bottom: 60px;
+      font-weight: bold;
+      font-size: 6.4rem;
+      line-height: 75px;
+      Color: @mainBg;
+    }
+  }
+
+  .cta {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    margin-top: 45px;
+
+
+  }
+
+  .sj-content {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto;
+
+    @media (max-width: 320px) {
+      flex-wrap: wrap;
+    }
+
```

Comment:
A @mobile variable was created in your variables.less. Why wasn't it utilized within your code?